### PR TITLE
Create confirmation dialog

### PR DESCRIPTION
**Pull request by e1121208**

### CODE REVIEW

**Comment by muhdbhz** [comment]
Can you push this to an upstream branch (not master) for bug testing?

**Review by muhdbhz — COMMENTED** [review]
Logic looks good but because of the FXML files, we cannot visualise this code without getting to run it personally. Pls see other comment

File: src/test/java/seedu/address/ui/ConfirmationDialogUiTest.java
Comment (lrongyi) [inline src/test/java/seedu/address/ui/ConfirmationDialogUiTest.java:1]: Tests LGTM

File: src/test/java/seedu/address/ui/ConfirmationDialogControllerUiTest.java
Comment (lrongyi) [inline src/test/java/seedu/address/ui/ConfirmationDialogControllerUiTest.java:1]: LGTM, seems like the confirmation dialog can be used with keyboard

File: src/test/java/seedu/address/logic/commands/DeleteClientProfileCommandTest.java [hunk 1]
```diff
@@ -92,32 +117,6 @@ public void execute_invalidNameFilteredList_throwsCommandException() {
 
         assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_PERSON_INPUT);
     }
-
```
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/DeleteClientProfileCommandTest.java:95]: Ok, tests are no longer needed as we no longer throw exceptions

File: src/main/java/seedu/address/logic/commands/DeleteClientProfileCommand.java [hunk 2]
```diff
@@ -49,13 +63,30 @@ public CommandResult execute(Model model) throws CommandException {
             }
         }
 
-        if (model.hasListingsForSeller(personToDelete)) {
-            throw new CommandException(String.format(MESSAGE_HAS_ACTIVE_LISTINGS, personToDelete.getName()));
+        if (!skipConfirmation
```
Comment (lrongyi) [inline src/main/java/seedu/address/logic/commands/DeleteClientProfileCommand.java:66]: Might violate code quality

**Pull request merged**